### DISCUSSION TITLE
Support dev dependencies in pom.xml

### DISCUSSION THREAD

**Discussion by metametadata** [aquasecurity/trivy#7346] [Ideas]
### Description

Vuln scanning of `pom.xml` should be able to scan dev dependencies too.

E.g. `--include-dev-deps` existing flag can be used to specify such behaviour.

Currently it's stated in https://aquasecurity.github.io/trivy/v0.54/docs/coverage/language/java/ that `pom.xml` dev deps will be excluded.

### Target

Filesystem

### Scanner

Vulnerability

**Comment by knqyf263** [comment]
Do you mean `<scope>test</scope>` should be included with `--include-dev-deps`?

**Reply by metametadata** [reply]
Yes.

**Reply by knqyf263** [reply]
It makes sense to me.
@DmitriyLewen What do you think? If it sounds reasonable to you, please convert it into an issue.

**Reply by DmitriyLewen** [reply]
yeah. It is good idea.

**Comment by DmitriyLewen** [comment]
Created #7384
